res/qml/Mixxx/qmldir

```diff
@@ -0,0 +1,8 @@
+module Mixxx
+linktarget mixxx-libplugin
+optional plugin mixxx-libplugin
+classname MixxxPlugin
+typeinfo mixxx-lib.qmltypes
+prefer :/mixxx.org/imports/Mixxx/
+PlayerDropArea 0.0 res/qml/Mixxx/PlayerDropArea.qml
+MathUtils 0.0 res/qml/Mixxx/MathUtils.mjs
```

Comment:
Qt does it the other way around: 
```suggestion
MathUtils 0.0 res/qml/Mixxx/MathUtils.mjs
PlayerDropArea 0.0 res/qml/Mixxx/PlayerDropArea.qml
```

Reply:
The order is irrelevant. But sure, we can sort it alphabetically...